### ISSUE TITLE
@K-E-N-W-A-Y  I have skillshare premium account can you halp us ?

### ISSUE THREAD

**Issue by NSbusiness** [K-E-N-W-A-Y/Skillshare-DL#14]

**Comment by jaydip430** [comment]
hey, I think @K-E-N-W-A-Y is long gone, he is offline. he might be having an exam or having so much work so let's try our own, can you give the cookies of your account?I just forked this repo

**Comment by NSbusiness** [comment]
Yah , But what will be my benefits ?

**Comment by jaydip430** [comment]
Brother thing about open source is you don't think about benefit. And this is piracy why people do privacy because they are insufficient to purchase or rent something

**Comment by jaydip430** [comment]
And thanks to open issue for premium account

**Comment by NSbusiness** [comment]
I will give you account can you help me by giving your new collab file

**Comment by jaydip430** [comment]
yep let's chat over telegram https://t.me/joinchat/b81h63o-FuQ5MTY1

**Comment by jaydip430** [comment]
this is new link for anyone t.me/skillshare_colab
or just type skillshare_colab in the telegram

**Comment by calvinhobbes23** [comment]
Enjoy people- I fixed this one:
https://colab.research.google.com/github/calvinhobbes23/Skillshare-DL/blob/master/Skillshare_DL_%5BKENWAY%5D.ipynb

 Thanks to someone who uploaded an account to the fileshare (although its a short term account)!

**Comment by PaxCaeli** [comment]
> 
> 
> Enjoy people- I fixed this one:
> https://colab.research.google.com/github/calvinhobbes23/Skillshare-DL/blob/master/Skillshare_DL_%5BKENWAY%5D.ipynb
> 
> Thanks to someone who uploaded an account to the fileshare (although its a short term account)!

thank you

**Comment by NSbusiness** [comment]
Can anybody tell me the process of fixing the colab and git ? I mean can
anybody share the process of fixing the problem or adding a new account in
git. I am new to git and also not an expert in programming.

**Comment by PaxCaeli** [comment]
> 
> 
> Enjoy people- I fixed this one:
> https://colab.research.google.com/github/calvinhobbes23/Skillshare-DL/blob/master/Skillshare_DL_%5BKENWAY%5D.ipynb
> 
> Thanks to someone who uploaded an account to the fileshare (although its a short term account)!

FYI for people that found this now: The account is now expired.

**Comment by jaydip430** [comment]
@PaxCaeli It's Working I Tested RIght Now

**Comment by kyoichishido** [comment]
> Enjoy people- I fixed this one:
> https://colab.research.google.com/github/calvinhobbes23/Skillshare-DL/blob/master/Skillshare_DL_%5BKENWAY%5D.ipynb
> 
> Thanks to someone who uploaded an account to the fileshare (although its a short term account)!

Hey Calvin, I have a favor to ask. I am a Comp sci sophomore and I tried to understand this program (for sake of learning) and didn't understand a lot. Would you please help me in understanding this code via a chat or meeting (anything you say) or cite some resources where I can learn or understand this program. Thankyou so much for fixing it, I am using it and it's still working. :)

Anyone, who'd be interested in explaining me this program please comment out here and would be grateful for your kindness.

**Comment by loolazoola** [comment]
> Enjoy people- I fixed this one:
> https://colab.research.google.com/github/calvinhobbes23/Skillshare-DL/blob/master/Skillshare_DL_%5BKENWAY%5D.ipynb
> 
> Thanks to someone who uploaded an account to the fileshare (although its a short term account)!

Thanks man
